bin/recipes/001-import.js

```diff
@@ -67,14 +68,21 @@ async.parallel({
 
 
 		// imported recipes for search data
-		helper.create(options, Recipe2, (err, data) => {
+		helper.create(options, Recipes2, (err, data) => {
```

Comment:
Recipe was declared as Recipes2, so I just added the "s"

Reply:
yep, i think i tried here to have an old version to work (aka Recipes Model) and tried to make works an updated version as Recipe2. Good approach! Maybe your current changes will force me to update the actual db architecture code. it was done 1y ago, so i think i start to know more about what should be improved here